mysql/CHANGELOG.md

```diff
@@ -45,3 +52,45 @@
 [#291]: https://github.com/DataDog/integrations-core/issues/291
 [#329]: https://github.com/DataDog/integrations-core/issues/329
 [#394]: https://github.com/DataDog/integrations-core/issues/394
+[#660]: https://github.com/DataDog/integrations-core/pull/660
```

Comment:
Sorry about that one, copy/paste failed me. Will be more careful on that. Should be good now